### ISSUE TITLE
Tabs always turn red on new notification no matter what.

### ISSUE THREAD

**Issue by xlJohnnyIcelx** [killjoy1221/TabbyChat-2#87]
Even when I disable this in the general settings the tabs still turn red when a new message appears in them. Is this just broken or is there something I'm missing?

**Comment by killjoy1221** [comment]
I see that the setting for `unreadFlashing` is actually never checked. Unfortunately, I don't remember where it is supposed to go. I believe I need to do a check when making is render. I will need to do that when I get a chance.

**Comment by xlJohnnyIcelx** [comment]
So if unread flashing is always off and it still turns red when a new message comes in then can you add a way to disable that as well? It would make this mod perfect for me

**Comment by killjoy1221** [comment]
Yes, that is what I'm getting at. This used to work, but I accidentally removed it a while ago.

**Comment by xlJohnnyIcelx** [comment]
Oh okay lol I'm glad I mentioned it then.
